File: backend/src/main/java/com/example/entity/NguoiDung.java
package com.example.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
@Entity
public class NguoiDung {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long maNguoiDung;
    private String hoVaTen;
    private Date ngaySinh;
    private String email;
    private String sdt;
    private String tenDangNhap;
    private String matKhau;
    private String vaiTro;
    private String diaChi;

    @ManyToOne
    @JoinColumn(name = "ma_doi_ngu", referencedColumnName = "maDoiNgu")
    private DoiNgu doiNgu;
}

File: fronted/src/models/NguoiDung.ts
import BaseResponse from "./BaseResponse";

export interface NguoiDungResponse extends BaseResponse {
    maNguoiDung: number
    hoVaTen: string
    ngaySinh: string
    email: string
    sdt: string
    tenDangNhap: string
    vaiTro: string
    diaChi: string
    doiNgu: string
}